specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/rbac.json

```diff
@@ -524,7 +523,7 @@
       "required": [
         "properties"
       ],
-      "description": "Role definition creation parameters."
+      "description": "Role definition create parameters."
```

Comment:
The old version seems to be more correct here - perhaps I am missing some context though.

Reply:
do you feel strongly? i can change it back - not sure that there was a reason for this, it's just how it read in 7.2 stable

Reply:
not really - it just reads strangely to me.